lms/djangoapps/ccx/views.py

```diff
@@ -367,17 +398,22 @@ def ccx_invite(request, course):
                 unenroll_email(ccx, email, email_students=email_students)
         except ValidationError:
             log.info('Invalid user name or email when trying to invite students: %s', email)
-    url = reverse('ccx_coach_dashboard', kwargs={'course_id': course.id})
+    url = reverse(
+        'ccx_coach_dashboard',
+        kwargs={'course_id': CCXLocator.from_course_locator(course.id, ccx.id)}
```

Comment:
This call indicates that my question about the definition of `ccx_coach_dashboard` is a reasonable one, since this is passing the `CCXLocator`, not the `CourseLocator`.